src/NuGet.Clients/NuGet.PackageManagement.UI/PackageInstallationInfo.cs

```diff
@@ -119,6 +119,11 @@ public override int GetHashCode()
             return StringComparer.OrdinalIgnoreCase.GetHashCode(_projectName);
         }
 
+        public override string ToString()
```

Comment:
Is this the text Narrator would read? What does the string look like when package is not installed and `InstalledVersion` is null? For the accessibility purpose I'd prefer using "Not installed" in that case.